databases/databases-for-redis/concepts.mdx

```diff
@@ -0,0 +1,117 @@
+---
+meta:
+  title: Databases for Redis™  - Concepts
+  description: This page explains all the concepts related to Databases for Redis™ .
+content:
+  h1: Concepts
+  paragraph: This page explains all the concepts related to Databases for Redis™ .
+tags: advanced settings
+categories:
+  - database
+---
+
+<Concept opened>
+
+## Availability Zone 
+
+An Availability Zone refers to the geographical location in which your instance will be created. The latency between multiple AZs of the same region is low, as they have a common network layer. Find below the list of availability zones by region:
+
+- France - Paris (`fr-par`)
+  `fr-par-1`
+  `fr-par-2`
+  `fr-par-3`
+- The Netherlands - Amsterdam (`nl-ams`)
+  `nl-ams-1`
+  `nl-ams-2` 
+- Poland - Warsaw (`pl-waw`)
+  `pl-waw-1`
+
+</Concept>
+
+<Concept opened>
+
+## Allowed IPs
+
+Access control list (ACL) rules [define permissions for remote access to a Database Instance](/databases/databases-for-postgresql-and-mysql/how-to/manage-allowed-ip-addresses/). A rule consists of an IP address or an IP range.
+
+</Concept>
+
+<Concept opened>
+
+## Cluster 
+
+A Redis™ cluster consists of a set of primary and secondary nodes. The cluster nodes use hash partitioning to split the keyspace into 16,384 key slots. Each primary of the cluster is responsible for a subset of those slots. Each secondary node replicates the data of one of the primary ones.
+
+</Concept>
+
+<Concept opened>
+
+## Cluster mode
+
+Each Managed Database for Redis™ cluster consists of three and up to six compute Instances hosting each a primary Redis™ Instance and a secondary Instance for one of the other nodes. If one of the primary nodes fails, the remaining nodes hold a vote and elect one of the remaining secondary nodes as the new primary node. When the failing node rejoins the cluster, it automatically becomes a secondary node and begins to replicate the data of another primary node. 
+
+</Concept>
+
+<Concept>
+
+## Endpoint
+
+A point of connection to a database. The endpoint is associated with one or more public IPv4 addresses and a port or with a private IPv4 and port (if you use [Private Networks]).
+
+</Concept>
+
+<Concept>
+
+## High availability
+
+High Availability is handled through the Redis™ clustering feature. This is a 3 nodes multi-master architecture. In case of downtime on a node in the cluster, a new node is able to take over the requests with minimal downtime.
```

Comment:
```suggestion
High Availability is handled through the Redis™ clustering feature. This is a 3-node multi-master architecture. In case of downtime on a node in the cluster, a new node is able to take over the requests with minimal downtime.
```